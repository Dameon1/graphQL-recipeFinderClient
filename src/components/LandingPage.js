import React from "react";
import { Link } from "@reach/router";

import { Landing } from "./styles";
import { LandingHeader } from "./styles";
import { LandingSmallPrint } from "./styles";
import { LandingText } from "./styles";
import { LandingButton } from "./styles";
import "./styles/landingPage.css";

export default function LandingPage() {
  return (
    <Landing>
      <LandingHeader>You Have Ingredients?</LandingHeader>

      <LandingSmallPrint> (Find recipes for those ingredients!) </LandingSmallPrint>

      <LandingText>Sign up today to enjoy full features</LandingText>

      <Link to="/signUp">
        <LandingButton>SignUp</LandingButton>
      </Link>

      <LandingText>or preview the experience</LandingText>

      <Link to="/dashboard">
        <LandingButton>Preview</LandingButton>
      </Link>
    </Landing>
  );
}
