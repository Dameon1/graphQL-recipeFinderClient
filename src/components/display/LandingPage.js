
import React from 'react';
import '../styles/landingPage.css';
import {Link} from 'react-router-dom';

export default function LandingPage() {

  return (
    <div className='landingPage'>
      <div className="landingPageBox">
        <h2 className='landingPageHeader'>You Have Ingredients?</h2>
        <p className='smallPrint'> (Find recipes for those ingredients!) </p>
        <p className="onBoarding">Sign up today to enjoy full features</p>
        <Link to='/signUp' >
          <button className="buttonStyle">SignUp</button>
        </Link>
        <p className="onBoarding">or preview the experience</p>
        <Link to='/dashboard' >
          <button className="buttonStyle">Preview</button>
        </Link>    
      </div>
    </div>
  )
};
