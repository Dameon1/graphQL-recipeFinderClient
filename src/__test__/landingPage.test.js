import React from "react";
import { shallow, mount } from "enzyme";
import LandingPage from "../components/display/landingPage";

describe("<LandingPage>", () => {
  it("renders without crashing", () => {
    shallow(<LandingPage />);
  });
});
