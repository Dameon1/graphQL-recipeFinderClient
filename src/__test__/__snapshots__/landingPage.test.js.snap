// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingPage> renders without crashing 1`] = `
<styled.div>
  <styled.h2>
    You Have Ingredients?
  </styled.h2>
  <styled.h2>
     
    (Find recipes for those ingredients!)
     
  </styled.h2>
  <styled.p>
    Sign up today to enjoy full features
  </styled.p>
  <ForwardRef
    to="/signUp"
  >
    <styled.button>
      SignUp
    </styled.button>
  </ForwardRef>
  <styled.p>
    or preview the experience
  </styled.p>
  <ForwardRef
    to="/dashboard"
  >
    <styled.button>
      Preview
    </styled.button>
  </ForwardRef>
</styled.div>
`;
